### DISCUSSION TITLE
keri-dev 12/07/23

### DISCUSSION THREAD

**Discussion by pfeairheller** [WebOfTrust/keri#59] [Meetings]
https://us06web.zoom.us/rec/share/jR1tRhpRqsi8xBekyDReB5v91QcRYbBbM4StqIT2I3euCy_qKLP3c6EplR2Q3jNU.0EINxV-_g_IJfAGE
Passcode: Eza$w!%4

### Action items
- None

#### filters
is:issue created:>2023-11-30
is:pr created:>2023-11-30

#### Issues:
- None

#### PRs
- https://github.com/WebOfTrust/keripy/pull/615
- https://github.com/WebOfTrust/keripy/pull/616
- https://github.com/WebOfTrust/keripy/pull/617
- https://github.com/WebOfTrust/keripy/pull/618
- https://github.com/WebOfTrust/keripy/pull/619
- https://github.com/WebOfTrust/keripy/pull/620
- https://github.com/WebOfTrust/keripy/pull/621
- https://github.com/WebOfTrust/keripy/pull/622
- https://github.com/WebOfTrust/keripy/pull/623
- https://github.com/WebOfTrust/keripy/pull/624
- https://github.com/WebOfTrust/keripy/pull/625
- https://github.com/WebOfTrust/keripy/pull/626

#### Issues
- https://github.com/WebOfTrust/keria/issues/142
- https://github.com/WebOfTrust/keria/issues/140
- https://github.com/WebOfTrust/keria/issues/141

#### PRs
- https://github.com/WebOfTrust/keria/pull/139
- https://github.com/WebOfTrust/keria/pull/143

#### Issues
- None

#### PRs
- https://github.com/WebOfTrust/signifypy/pull/100

#### Issues
- https://github.com/WebOfTrust/signify-ts/issues/155
- https://github.com/WebOfTrust/signify-ts/issues/159
- https://github.com/WebOfTrust/signify-ts/issues/164
- https://github.com/WebOfTrust/signify-ts/issues/165

#### PRs
- https://github.com/WebOfTrust/signify-ts/pull/157
- https://github.com/WebOfTrust/signify-ts/pull/160
- https://github.com/WebOfTrust/signify-ts/pull/162
- https://github.com/WebOfTrust/signify-ts/pull/167
- https://github.com/WebOfTrust/signify-ts/pull/168
- https://github.com/WebOfTrust/signify-ts/pull/169
- https://github.com/WebOfTrust/signify-ts/pull/152
- https://github.com/WebOfTrust/signify-ts/pull/153
- https://github.com/WebOfTrust/signify-ts/pull/154
- https://github.com/WebOfTrust/signify-ts/pull/156
- https://github.com/WebOfTrust/signify-ts/pull/158
- https://github.com/WebOfTrust/signify-ts/pull/161
- https://github.com/WebOfTrust/signify-ts/pull/163
- https://github.com/WebOfTrust/signify-ts/pull/166

#### Issues
- https://github.com/WebOfTrust/WOT-terms/issues/79
- https://github.com/WebOfTrust/WOT-terms/issues/81
- https://github.com/WebOfTrust/WOT-terms/issues/87
- https://github.com/WebOfTrust/WOT-terms/issues/82
- https://github.com/WebOfTrust/WOT-terms/issues/83
- https://github.com/WebOfTrust/WOT-terms/issues/84
- https://github.com/WebOfTrust/WOT-terms/issues/85
- https://github.com/WebOfTrust/WOT-terms/issues/86

#### PRs
- https://github.com/WebOfTrust/WOT-terms/pull/80

#### Issues
- None

#### PRs
- https://github.com/WebOfTrust/cesr-ts/pull/15
- https://github.com/WebOfTrust/cesr-ts/pull/16

#### Issues
- https://github.com/WebOfTrust/cesride/issues/160

#### PRs
- https://github.com/WebOfTrust/cesride/pull/161

#### Issues
- None

#### PRs
- None

**Comment by lenkan** [comment]
This is the PR for chained credential integration test https://github.com/WebOfTrust/signify-ts/pull/144, it is a couple of weeks old so it is not in the list above.

**Comment by pfeairheller** [comment]
### Discussion

#### Signify Browser Extension
- Poll for name: https://github.com/WebOfTrust/signify-browser-extension/discussions/1
- Rodo Presentation - See recording
